tools/jbrowse2/jbrowse2.xml

```diff
@@ -0,0 +1,873 @@
+                            <expand macro="input_conditional" label="BED Track Data" format="bed"/>
+                            <expand macro="track_styling_feature"/>
+                            <expand macro="track_visibility"/>
+                        </when>
+                        <when value="cram">
+                            <expand macro="input_conditional" label="CRAM Track Data" format="cram"/>
+                            <expand macro="track_visibility"/>
+                        </when>
+                        <when value="maf">
+                            <expand macro="input_conditional" label="MAF Track Data" format="maf"/>
+                            <expand macro="track_styling_feature"/>
+                            <expand macro="track_visibility"/>
+                        </when>
+                        <when value="bigwig">
+                            <expand macro="input_conditional" label="BigWig Track Data" format="bigwig"/>
+                            <expand macro="track_visibility"/>
+                        </when>
+                        <when value="paf">
+                            <expand macro="input_conditional" label="PAF format synteny mapping data" format="paf" help="PAF made with mashmap or minimap2 mapping the real reference as a query on the comparison genomes as references"/>
+                            <expand macro="pafref_conditional" label="Comparison genome sequence(s)" format="fasta" help="Paf from these as the reference(s), using the common reference as the reads to map"/>
+                            <expand macro="track_visibility"/>
+                        </when>
+                        <when value="hic">
+                            <expand macro="input_conditional" label="Binary Juicebox HiC data" format="hic,juicebox_hic"/>
+                            <expand macro="track_visibility"/>
+                        </when>
+                        <when value="cool">
+                            <expand macro="input_conditional" label="HiC data in cool/mcool/scool format" format="cool,mcool,scool"/>
+                            <expand macro="track_visibility"/>
+                        </when>
+                    </conditional>
+                </repeat>
+            </repeat>
+        </repeat>
+        <expand macro="general_options"/>
+        <param type="hidden" name="uglyTestingHack" value=""/>
+    </inputs>
+    <outputs>
+        <data format="html" name="output" label="JBrowse2">
+            <change_format>
+                <when input="zipOut" value="true" format="zip"/>
+            </change_format>
+        </data>
+    </outputs>
+    <tests>
+        <test>
+            <repeat name="assemblies">
+                <conditional name="reference_genome">
+                    <param name="genome_type_select" value="history"/>
+                    <param name="genome" value="merlin.fa"/>
+                    <param name="genome.ext" value="fasta"/>
```

Comment:
I would recommend to use `genome_ext`, because `genome.ext` could be confusing with the cheetah syntax of nested params.

Reply:
yes please!